components/CalendarExportModal/CalendarExportModal.tsx

```diff
@@ -0,0 +1,164 @@
+import { Modal, Box, Typography, Fade, Backdrop } from "@mui/material";
+
+import { useState } from "react";
+
+import { Collapse } from "antd";
+
+type CalendarExportModalProps = {
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+  isHome: boolean;
+};
+
+const CalendarExportModal = ({
+  isOpen,
+  setIsOpen,
+  isHome,
+}: CalendarExportModalProps) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  function copyToClipboard() {
+    navigator.clipboard.writeText(
+      "https://calendarium.cesium.di.uminho.pt/api/export?subjects=221,222" // TODO: function that generates actual URL
```

Comment:
We should create a .env file with a `DOMAIN` variable. Moreover, we should have a .env.dev (for local development) and a .env.prod (for production deployment) where the variable value would be `localhost:3000` and `calendarium.cesium.di.uminho.pt`, respectively.

Reply:
done